docs/02_guides/pay_per_event.mdx

```diff
@@ -0,0 +1,46 @@
+---
+id: pay-per-event
+title: Pay-per-event Monetization
+description: Monetize your Actors using the pay-per-event pricing model
+---
+
+import ActorChargeSource from '!!raw-loader!./code/actor_charge.py';
+import ConditionalActorChargeSource from '!!raw-loader!./code/conditional_actor_charge.py';
+import ApiLink from '@site/src/components/ApiLink';
+import CodeBlock from '@theme/CodeBlock';
+
+Apify provides several [pricing models](https://docs.apify.com/platform/actors/publishing/monetize) for monetizing your Actors. The most recent and most flexible one is [pay-per-event](https://docs.apify.com/platform/actors/running/actors-in-store#pay-per-event), which lets you charge your users programmatically directly from your Actor. As the name suggests, you may charge the users each time a specific event occurs, for example a call to an external API or when you return a result.
+
+To use the pay-per-event pricing model, you first need to [set it up](https://docs.apify.com/platform/actors/running/actors-in-store#pay-per-event) for your Actor in the Apify console. After that, you're free to start charging for events.
+
+## Charging for events
+
+After monetization is set in the Apify console, you can add <ApiLink to="class/Actor#charge">`Actor.charge`</ApiLink> calls to your code and start monetizing!
```

Comment:
Locally this points to http://localhost:3000/sdk/python/api/class/Actor, but the correct link is http://localhost:3000/sdk/python/reference/class/Actor, is that expected not to work locally? I don't think so, as in Crawlee it works.